frontend/src/pages/answer/fullKg/simulation.worker.js

```diff
@@ -0,0 +1,21 @@
+/** Web Worker to run d3 force simulation and return placed nodes and edges */
```

Comment:
How does this work? In KgFull.jsx we import Worker from `./simulation.worker`, but there is no Worker defined here as far as I can tell.

Reply:
This might be specific to webpack as you can see from these [examples](https://webpack.js.org/loaders/worker-loader/#examples). It knows this is a web worker file from the `.worker.js` extension. In vanilla JS, you would call it as `const worker = new Worker(<path/to/worker/file>);`

Reply:
This might be specific to webpack (see [examples](https://webpack.js.org/loaders/worker-loader/#examples)) but webpack basically knows this is a web worker file and to auto-initialize it as a `Worker`. The same thing in vanilla javascript would be `const worker = new Worker(<path/to/webworkerfile.js>);`.